Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/ConfigChangeDetector.kt

```diff
@@ -60,4 +64,19 @@ class ConfigChangeDetector @Inject constructor(
             RiskLevelRepository.setRiskLevelScore(RiskLevel.UNDETERMINED)
         }
     }
+
+    @VisibleForTesting
+    internal suspend fun checkForDiagnosisKeysDataMapping(exposureWindowRiskCalculationConfig: ExposureWindowRiskCalculationConfig) {
+        val oldDiagnosisKeyDataMapping = enfClient.getDiagnosisKeysDataMapping()
```

Comment:
This is the _current_ diagnosisKeysDataMapping, right? Might be worth adjusting the variable name accordingly 😉

Reply:
fixed in new place